src/sentry/coreapi.py

```diff
@@ -174,16 +174,16 @@ def __init__(self, agent=None, version=None, project_id=None,
         self.log = ClientLogHelper(self.context)
 
     def auth_from_request(self, request):
-        if request.META.get('HTTP_X_SENTRY_AUTH', '').startswith('Sentry'):
+        if request.META.get('HTTP_X_SENTRY_AUTH', '')[:7] == 'Sentry ':
```

Comment:
Looks good to me. I would .lower() it though. HTTP tokens are in theory case insensitive. Would make sense to support.